lib/unittest.bzl

```diff
@@ -32,6 +32,15 @@ def register_unittest_toolchains():
 
 TOOLCHAIN_TYPE = "@bazel_skylib//toolchains/unittest:toolchain_type"
 
+# A private attribute used to denote a test rule is an analysis test.
+# For backwards compatibility, "unit tests" are not analysis tests, as being
+# an analysis test enforces the following test restrictions:
+#
+# 1. Test implementation functions may not register their own adhoc actions.
+# 2. There is a hard limit on the number of allowed transitive dependencies of
+#    analysis-test targets.
+_IS_ANALYSIS_TEST_ATTR_NAME = "_is_analysis_test"
```

Comment:
Is this some constant that also exists in bazel and has to match it?

Reply:
It is not. I just wanted to pull a constant out within this file and explain its significance

Reply:
(But note now this constant is no longer needed)